meson.build

```diff
@@ -24,16 +24,12 @@ endif
 
 # How to disable OpenMP:
 # The 'SILX_WITH_OPENMP' file should contain 'False'
-if os.is_file('SILX_WITH_OPENMP')
-  openmp_var = os.read('SILX_WITH_OPENMP').strip()
+res = run_command(py, '-c', 'import os; print(os.environ["SILX_WITH_OPENMP"])',
```

Comment:
Removing file-based setting as proposed in https://github.com/silx-kit/silx/pull/4277#discussion_r2077005790